assets/js/util.js

```diff
@@ -7,16 +7,6 @@ var SPACE_KEY_CODE = 32, ENTER_KEY_CODE = 13,
     HTTP_OK_CODE = 200, HTTP_BAD_REQUEST_CODE = 400,
     XHR_LOADING = 3, XHR_DONE = 4;
 
-var TEXTAREA_AUTO_RESIZE_MINIMUM_WIDTH = 768,
-    BACK_TO_TOP_BUTTON_ACTIVATION_HEIGHT = 300,
-    APY_REQUEST_URL_THRESHOLD_LENGTH = 2000, // maintain 48 characters buffer for generated parameters
```

Comment:
I don't know what the comment is about but you should probably preserve it in the config file.

Reply:
Sure.

Reply:
I think `2000 = 2^11 - 48` btw.

Reply:
I don't actually think it should be configurable.

Reply:
Okay